<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPracticeDialog</class>
 <widget class="QDialog" name="AddPracticeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddPracticeDialog</string>
  </property>
  <widget class="PracticeSettings" name="practiceSettings" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>721</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="PracticeDoctorSettings" name="doctorSettings" native="true">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>411</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>480</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>practiceSettings</zorder>
  <zorder>okButton</zorder>
  <zorder>doctorSettings</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PracticeSettings</class>
   <extends>QWidget</extends>
   <header>View/SettingsDialog/PracticeSettings/PracticeSettings.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PracticeDoctorSettings</class>
   <extends>QWidget</extends>
   <header>View/SettingsDialog/PracticeDoctorSettings/practicedoctorsettings.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
